src/Npgsql/Properties/NpgsqlStrings.Designer.cs

```diff
@@ -158,5 +158,11 @@ internal class NpgsqlStrings {
                 return ResourceManager.GetString("FullTextSearchNotEnabled", resourceCulture);
             }
         }
+        
+        internal static string WriteRecordNotSupported {
```

Comment:
It's nice but in normal use it'll never be seen, unless supportsWriting was not passed correctly in the info.

Reply:
Right, I see Write_Record_is_not_supported failing on that now. Would it better to removed supportsWriting altogether and to have converters throw instead?

Reply:
No because converters don't get the chance to run during resolution, which is where this error should be thrown